src/core/server/http/base_path_proxy_server.test.ts

```diff
@@ -66,7 +66,7 @@ describe('BasePathProxyServer', () => {
     config = ({
       name: 'kibana',
       host: '127.0.0.1',
-      port: 10002,
+      port: 10012,
```

Comment:
note to myself: use `get-port` here

Reply:
Yeah, you probably noticed, but I think the tests are running concurrently and when I had the same port numbers as other tests they ended up failing so I had to change it. I think if you have a way to dynamically assign unused ports that would be better for these types of tests.